alkira/resource_alkira_service_pan.go

```diff
@@ -41,13 +41,53 @@ func resourceAlkiraServicePan() *schema.Resource {
 				Type:        schema.TypeString,
 				Required:    true,
 			},
+			"global_protect_enabled": {
+				Description: "Enable global protect option or not. Default is `false`",
+				Type:        schema.TypeBool,
+				Required:    true,
+			},
+			//TODO(mac): finish writing this descritpion
```

Comment:
Could we finish this before releasing? Also, it's probably good to say that global_protect_segment_option will need global_protect to be enabled.

Reply:
Yes! I put this here to remind me to write a better description and forgot. Looks like it did it's job!